configuration/scripts/options/set_nml.snw30percent

```diff
@@ -1,7 +1,17 @@
 tr_snow = .true.
+snwgrain        = .true.
```

Comment:
snwgrain and use_smliq_pnd should not be set here - my mistake

Reply:
Both set_nml.nsw30percent and set_nml.nswITDrdg control wind redistribution of snow, which is fundamentally a separate process from the snow grain metamorphism.  What I had in mind is to be able to set the processes individually (with others turned off) as -s snw30percent or -s snwITDrdg, or together as e.g. -s snwITDrdg,snwgrain.  So I don't think we should have snw_aging_table etc in the snow redistribution options, but it should be in snwgrain.  In the case where we want to run both, that should set snw_aging_table appropriately regardless of the order -s snwITDrdg,snwgrain or -s snwgrain,snwITDrdg, right?